tests/test_os_utils.py

```diff
@@ -38,11 +39,12 @@ def test_get_stdout_from_command():
     assert stdout == ['asdf']
 
 
+@pytest.mark.skipif(not sys.platform.startswith('darwin'),
```

Comment:
does error look different in case of other operating systems? can we check something else like a part of the string is present or something in the error output for other systems